pixi.toml

```diff
@@ -0,0 +1,153 @@
+[project]
+name = "plateau"
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "osx-64", "linux-64", "linux-aarch64", "win-64"]
+
+[tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
+
+[dependencies]
+python = ">=3.10"
+dask = ">=2022.5.1"
+decorator = "*"
+msgpack-python = ">=0.5.2"
+numpy = ">1.23,<2"
+pandas = ">1.5.0"
+pyarrow = ">=4.0.1"
+simplejson = "*"
+minimalkv = ">=1.4.2"
+toolz = "*"
+urlquote = ">=1.1.3"
+zstandard = "*"
+attrs = "*"
+click = "*"
+prompt-toolkit = "*"
+pyyaml = "*"
+
+
+[host-dependencies]
+pip = "*"
+setuptools = ">=61"
+setuptools-scm = "*"
+
+[feature.test.dependencies]
+pytest = ">=6"
+pytest-cov = "*"
+pytest-mock = "*"
+pytest-xdist = ">=3.6.1,<4"
+freezegun = "*"
+mypy = "*"
+distributed = ">=2022.5.1"
+pytz = "*"
+cloudpickle = "*"
+hypothesis = "*"
+[feature.test.tasks]
+test = "pytest"
+test-coverage = "pytest --cov=plateau --cov-report=xml --cov-report=term-missing"
+
+[feature.build.dependencies]
+python-build = "*"
+twine = "*"
+[feature.build.tasks]
+build-wheel = "python -m build --no-isolation ."
+check-wheel = "twine check dist/*"
+
+[feature.lint.dependencies]
+pre-commit = "*"
+insert-license-header = "*"
+docformatter = "*"
+ruff = "*"
+prettier = "*"
+taplo = "*"
+pre-commit-hooks = "*"
+typos = "*"
+[feature.lint.tasks]
+pre-commit-install = "pre-commit install"
+pre-commit-run = "pre-commit run -a"
+
+[feature.benchmark.dependencies]
+asv = "*"
+libmambapy = "*"
+conda = "*"
+
+[feature.docs.dependencies]
+sphinx = "*"
+jinja2 = "<3.1"
+sphinx_rtd_theme = "*"
+IPython = "*"
+[feature.docs.tasks]
+# Run task `postinstall` before building docs in docs environment
+docs = "cd docs && make html"
+readthedocs = "rm -rf $READTHEDOCS_OUTPUT/html && cp -r docs/_build/html $READTHEDOCS_OUTPUT/html"
+
+
+[feature.py310-pyarrow4_0_1.dependencies]
+python = "3.10.*"
+pyarrow = "=4.0.1"
+[feature.py310-pyarrow7_0.dependencies]
+python = "3.10.*"
+pyarrow = "=7.0.0"
+[feature.py310-pyarrow8_0_1.dependencies]
+python = "3.10.*"
+pyarrow = "=8.0.1"
+[feature.py310-pyarrow9_0.dependencies]
+python = "3.10.*"
+pyarrow = "=9.0.0"
+[feature.py310-pyarrow_nightly.dependencies]
+python = "3.10.*"
+# let's see how this can be done, perhaps via pip
+
+[feature.py311-pyarrow10_0_1.dependencies]
+python = "3.11.*"
+pyarrow = "=10.0.1"
+[feature.py311-pyarrow11_0.dependencies]
+python = "3.11.*"
+pyarrow = "=11.0.0"
+[feature.py311-pyarrow12_0.dependencies]
+python = "3.11.*"
+pyarrow = "=12.0.0"
+[feature.py311-pyarrow13_0.dependencies]
+python = "3.11.*"
+pyarrow = "=13.0.0"
+[feature.py311-pyarrow13_0-pandas1_5_3.dependencies]
+python = "3.11.*"
+pyarrow = "=13.0.0"
+pandas = "=1.5.3"
```

Comment:
Do we still need this pinning?

According to spec 0, support for pandas 1.5.0 can be dropped

Reply:
No, you can drop a lot here. Please move this to the oldest SPEC0-support pandas version.